pkg/clients/types/errors.go

```diff
@@ -13,8 +13,25 @@
  *******************************************************************************/
 package types
 
+import "fmt"
+
 type ErrNotFound struct{}
 
 func (e ErrNotFound) Error() string {
 	return "item not found"
 }
+
+type ErrClientServices struct{
```

Comment:
@brandonforster While you're in there, I'm thinking "ErrServiceClient" is a better name for this indicating that the error is coming from one of our service client wrappers. If you agree, can you rename?